content/en/user-guide/aws/events/index.md

```diff
@@ -1,10 +1,7 @@
 ---
 title: "EventBridge"
 linkTitle: "EventBridge"
-description: >
-  Get started with EventBridge on LocalStack
-aliases:
-- /user-guide/aws/eventbridge/
```

Comment:
Why do we remove the alias here?

Reply:
No longer needed. They were added in Dec '22 during migrating docs to a new structure. The Search result links are now up-to-date.